runatlantis.io/docs/webhook-secrets.md

```diff
@@ -25,8 +25,8 @@ To mitigate, use repo whitelists and IP whitelists. See [Security](security.html
 You can use any random string generator to create your Webhook secret. It should be > 24 characters.
 
 For example:
-* Generate via Ruby with `ruby -rsecurerandom -e 'puts SecureRandom.hex(20)'`
-* Generate online with [https://www.random.org/passwords/?num=2&len=20&format=html&rnd=new](https://www.random.org/passwords/?num=2&len=20&format=html&rnd=new)
+* Generate via Ruby with `ruby -rsecurerandom -e 'puts SecureRandom.hex(32)'`
+* Generate online with [https://www.random.org/passwords/?num=2&len=16&format=html&rnd=new](https://www.random.org/passwords/?num=2&len=16&format=html&rnd=new)
```

Comment:
can you leave this as 20? It was better as 20 than 16.

Reply:
sure